css-fonts/styles.css

```diff
@@ -0,0 +1,7 @@
+h1,
+h2,
+p {
+  font-family: 'Roboto Condensed', sans-serif;
+  line-height: 30px;
+  /* font-family: 'Oswald', sans-serif; */
+}
```

Comment:
Looks like you're only applying 'Roboto Codensed' here, and you're applying it to everything. You should have a ruleset just for your `h1` & `h2`, where you apply "Roboto Condensed" for example, and then you need another ruleset for your `p` where you apply the other `font-family`